release/release.md

```diff
@@ -103,20 +103,32 @@ $ git pull
    $./gen_versions_txt.sh ${BRANCH}
    $PUSH=1 OBS_SUBPROJECT="releases:$(uname -m):${BRANCH}" ./build_from_docker.sh ${NEW_VERSION}
    ```
+
+6. Test packages
+After the all the packages built succesfully (see status in OBS web page: https://build.opensuse.org/project/subprojects/home:katacontainers),
+make sure the packages work for installation and fuctional. To help with this you can use the package test jobs.
+
+- http://jenkins.katacontainers.io/job/package-release-testing/build?delay=
```

Comment:
This URL is invalid and gives me an amusing "Jenkins Demon" ;) Should it just be `http://jenkins.katacontainers.io/job/package-release-testing/build`?

Also, that job *appears* to be https://github.com/kata-containers/ci/blob/master/jenkins/jobs/release/config.xml [1]? I'm finding that XML rather hard to understand: where is that `<script/>` section installing the latest OBS packages?

---

[1] - I say appears because I cannot find any mention of `package-release-testing` in any of the repos - have the latest changes been pushed to https://github.com/kata-containers/ci?

Reply:
@jodh-intel true, it fails the real URL is http://jenkins.katacontainers.io/job/package-release-testing/

Reply:
It is not backup in ci repository yet,  it is not ready IMO,  installation was hardcoded to change the  branch, I prefer use katamanager but first need to allow change the branch from it.

https://github.com/kata-containers/documentation/pull/424/files

This will help for that

Reply:
URL fixed